### DISCUSSION TITLE
这条怎么改呢？每次切换都报错😵‍💫

### DISCUSSION THREAD

**Discussion by Dominiquema** [vernesong/OpenClash#2743] [Q&A]
![E1B99E89-6504-4E15-99F3-901C2D38E620](https://user-images.githubusercontent.com/108907039/195689437-cf7cde21-2d74-4aef-8628-4d07ef52de53.png)

**Comment by shizongrengkou** [comment]
你这个应该是配置文件没写对，下面的规则有问题那不是报错了吗？自己看看试试没改规则代理组的名字
